Runtime/UIElements/Controls/UIUtils.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+static class UIUtils
+    {
+        static readonly string[] s_SizeUnits = { "KB", "MB", "GB", "TB" };
+
+        public static void SetElementDisplay(VisualElement element, bool value)
```

Comment:
I am not sure why do we need this.
If we have `DisplayStyle.None` there is no point to also apply `Visibility.Hidden`

I might be missing something tho.